### ISSUE TITLE
Missing user badges

### ISSUE THREAD

**Issue by artch** [NodeBB/NodeBB#5865]
- **NodeBB version:** 1.5.3
- **NodeBB git hash:** f6890cd971e5b66752d1073ac523b74af4cd9588
- **Database type:** mongo
- **Database version:** v3.4.6

Some user badges are missing despite their group membership is fine. I've uploaded a video displaying this: https://www.dropbox.com/s/tegjft41yefy2dp/2017-08-09_16-52-30.mp4?dl=0

Notice that there is no badge neither on posts lists page nor in user profile, but the group is set and "Show Badge" option is on.

What info should I provide to debug this issue?

**Comment by vedi** [comment]
I guess, it should be related to "Group title" in user profile.

**Comment by barisusakli** [comment]
I tried to reproduce this but it worked for me, here are the steps I took.

1. Create group names `[Alliance] Overlords`
2. Change group badge title to `Overlords`
3. Add a new user to group `[Alliance] Overloards`
4. Check user profile to see badge.

Are your repro steps different?

Another thing to keep in mind, a user automatically gets the group title only if they haven't set a groupTitle before. If they have another value set in `groupTitle` they won't automatically get the new one.
